recipes/ogre/1.x/conanfile.py

```diff
@@ -0,0 +1,320 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import conan.tools.files
+import textwrap, shutil
+import functools
+
+class ogrecmakeconan(ConanFile):
+    name = "ogre"
+    license = "MIT"
+    homepage = "https://github.com/OGRECave/ogre"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "A scene-oriented, flexible 3D engine written in C++ "
+    topics = ("graphics", "rendering", "engine", "c++")
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    generators = "cmake", "cmake_find_package"
+    exports_sources = "CMakeLists.txt", "patches/**"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "cp_bin_dir": "ANY",
+        "cp_media_dir": "ANY",
+        "disable_plugins": [True, False],
+        "ogre_assert_mode": "ANY",
+        "ogre_build_component_bites": [True, False],
+        "ogre_build_component_hlms": [True, False],
+        "ogre_build_component_meshlodgenerator": [True, False],
+        "ogre_build_component_overlay": [True, False],
+        "ogre_build_component_paging": [True, False],
+        "ogre_build_component_property": [True, False],
+        "ogre_build_component_python": [True, False],
+        "ogre_build_component_rtshadersystem": [True, False],
+        "ogre_build_component_terrain": [True, False],
+        "ogre_build_component_volume": [True, False],
+        "ogre_build_dependencies": [True, False],
+        "ogre_build_plugin_bsp": [True, False],
+        "ogre_build_plugin_octree": [True, False],
+        "ogre_build_plugin_pcz": [True, False],
+        "ogre_build_plugin_pfx": [True, False],
+        "ogre_build_rendersystem_d3d11": [True, False],
+        "ogre_build_rendersystem_gl": [True, False],
+        "ogre_build_rendersystem_gl3plus": [True, False],
+        "ogre_build_samples": [True, False],
+        "ogre_build_tests": [True, False],
+        "ogre_build_tools": [True, False],
+        "ogre_config_enable_quad_buffer_stereo": [True, False],
+        "ogre_config_filesystem_unicode": [True, False],
+        "ogre_config_threads": "ANY",
+        "ogre_config_thread_provider": "ANY",
+        "ogre_config_enable_freeimage": [True, False],
+        "ogre_install_pdb": [True, False],
+        "ogre_install_samples": [True, False],
+        "ogre_install_tools": [True, False],
+        "ogre_install_vsprops": [True, False],
+        "ogre_resourcemanager_strict": "ANY",
+        "ogre_static": [True, False],
+        "ogre_set_double": [True, False],
+        "ogre_glsupport_use_egl": [True, False]
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "cp_bin_dir": "bin",
+        "cp_media_dir": "Media",
+        "disable_plugins": False,
+        "ogre_assert_mode": 1,
```

Comment:
I know all official options have the prefix ogre_, but I think it could be removed, as you need to pass the package name for any option when running a conan command. E.g. `ogre::build_component_bites=True`

Reply:
sounds good. Will do the changes